server.js

```diff
@@ -39,7 +39,8 @@ function devOrHeroku() {
 }
 
 
-if (app.get("env") !== "dev") {
+// dev, heroku, and stage environments don't need to force https for all urls
+if (!["dev", "heroku", "stage"].includes(app.get("env"))) {
```

Comment:
Why stage? Shouldn't stage and prod be virtually identical?

Reply:
Good question, and we may hit a similar issue in prod. Hmm ... I may have to dig into why removing `trust proxy` seemed to break the Dockerflow heartbeat routes.